tcms/testruns/models.py

```diff
@@ -268,14 +266,24 @@ class TestExecutionStatus(TCMSActionModel):
         'FAIL': '#cc0000'
     }
 
-    complete_status_names = (PASSED, 'ERROR', FAILED, WAIVED)
-    failure_status_names = ('ERROR', FAILED, BLOCKED)
-    passed_status_names = (PASSED, WAIVED)
-    idle_status_names = (IDLE,)
+    from django.utils.translation import gettext_lazy as _
+    POSITIVE = 'P', _('Positive')
+    NEGATIVE = 'N', _('Negative')
+    NEUTRAL = 'NN', _('Neutral')
+
     chart_status_names = ('pass', 'fail', 'idle', 'other')
 
     id = models.AutoField(db_column='case_run_status_id', primary_key=True)
     name = models.CharField(max_length=60, blank=True, unique=True)
+    status = models.CharField(
+        max_length=60,
+        choices=[POSITIVE, NEGATIVE, NEUTRAL]
+    )
+    is_completed = models.BooleanField()
+    chart_name = models.CharField(max_length=60)
+    icon = models.CharField(name="icon", max_length=60)
+    color = models.CharField(name="color", max_length=60)
+    color_code = models.CharField(name="color_code", max_length=60)
```

Comment:
`color` is just fine, we don't need 2 of them. Also hex colors are only 6 chars so you can limit the text field to 8 chars. Even better this is an actual hex number and it can fit inside an integer field, the input/output is just different.

Also see 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color
https://stackoverflow.com/questions/39859224/how-to-use-html5-color-picker-in-django-admin/39864918
https://github.com/jaredly/django-colorfield